prov/rxd/src/rxd_cq.c

```diff
@@ -1064,6 +1065,7 @@ static void rxd_handle_start_data(struct rxd_ep *ep, struct rxd_peer *peer,
 repost:
 	rxd_ep_repost_buff(rx_buf);
 out:
+	assert(rxd_reposted_bufs);
```

Comment:
this code is executed inside of locks.
there are jumps to "out" label, without re-post buffer
my idea is catch cases when no buffer are reposted, I spent few hours looking why test hangs :)